nautobot/ipam/tables.py

```diff
@@ -62,10 +62,30 @@
 {% endif %}
 """
 
+IPADDRESS_COPY_LINK = """
```

Comment:
Since you're in here, this seems like a good opportunity to move away from `template_code` and instead use `template_name` so that these template snippets can be stored as template files instead of hard-coded template strings.

Perhaps it's out of scope for this, but inevitably is something we need to address for the UI overhaul.

Reply:
Thanks @jathanism. @bryanculver  can you please let me know how I can create template_name?